pkgs/servers/gotify/ui.nix

```diff
@@ -1,12 +1,17 @@
 { yarn2nix-moretea
 , fetchFromGitHub, applyPatches
+, fetchYarnDeps
 }:
 
 yarn2nix-moretea.mkYarnPackage rec {
   pname = "gotify-ui";
 
   packageJSON = ./package.json;
-  yarnNix = ./yarndeps.nix;
+
+  offlineCache = fetchYarnDeps {
```

Comment:
Hey and thanks for the patch! I always wondered, why does `mkYarnPackage` needs both `packageJSON`, and `offlineCache`? If it already relies on `${src}` to prepare a fixed-output derivation, why can't it also parse `"${src}/package.json"`?

Reply:
using `${src}` to prepare a fixed-output derivation does not lead to IFD, since the out path of the src derivation is known without building it.

When parsing `"${src}/package.json"`, it would not only have to know the out path where the file will land, but also the contents, hence it would need to build the derivation before continuing with evaluation.

Reply:
I will take care of the update script

Long-term, the solution is to deprecate mkYarnPackage, and instead introduce a yarnConfigHook similar to npmConfigHook